extensions/oidc-db-token-state-manager/deployment/src/main/java/io/quarkus/oidc/db/token/state/manager/OidcDbTokenStateManagerBuildTimeConfig.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.oidc.db.token.state.manager;
+
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+@ConfigMapping(prefix = "quarkus.oidc-db-token-state-manager")
```

Comment:
Michal, can we try `quarkus.oidc.db-token-state-manager` - it should align better with the rest of the OIDC configuration

Reply:
yes, I originally had it as well, but wasn't sure if conventions allow it (e.g. quarkus.oidc.* should belong to oidc extension). I agree.

I'll wait for a day if there comes another review regarding db stuff and change it. Thanks